awx/settings/defaults.py

```diff
@@ -1052,4 +1053,6 @@
 # Host metrics cleanup - minimal interval between two cleanups in days
 CLEANUP_HOST_METRICS_INTERVAL = 30  # days
 # Host metrics cleanup - soft-delete HostMetric records with last_automation < [threshold] (in months)
-CLEANUP_HOST_METRICS_THRESHOLD = 12  # months
+CLEANUP_HOST_METRICS_SOFT_THRESHOLD = 12  # months
+# Host metrics cleanup - delete HostMetric record with deleted=True and last_deleted < [threshold]
```

Comment:
In practice, I ran `awx-manage host_metric_summary_monthly` and it created 36 records (going back to 2020), which is determined by this setting. I can't get from what this comment says to this behavior.